src/Processors/QtiV2/In/Interactions/TextEntryInteractionMapper.php

```diff
@@ -25,6 +47,35 @@ public function getQuestionType()
         return $closetext;
     }
 
+    public function getOption($key, $defaultValue = null) {
+        $value = $defaultValue;
+        if (isset($this->options[$key])) {
+            $value = $this->options[$key];
+        }
+
+        return $value;
+    }
+
+    public function isUsingExpectedLengthAsMaxCharLimit() {
+        return (bool)$this->getOption('expected_length_as_max_char_limit', false);
+    }
+
+    public function isUsingExpectedLengthAsResponseWidth() {
+        return (int)$this->getOption('expected_length_as_response_width', true);
```

Comment:
An isser should return a boolean. See previous note re coercing the return type. If it needs to return an int, lets change it to a getter.

Reply:
Wow this line made no sense. Yeah it should be a boolean, not sure what part of my brain thought otherwise.